aviary/examples/external_subsystems/engine_NPSS/table_engine_builder.py

```diff
@@ -0,0 +1,183 @@
+import openmdao.api as om
+
+from aviary.subsystems.propulsion.engine_model import EngineModel
+from aviary.utils.aviary_values import AviaryValues
+
+from aviary.examples.external_subsystems.engine_NPSS.NPSS_Model.DesignEngineGroup import DesignEngineGroup
+from aviary.examples.external_subsystems.engine_NPSS.engine_variable_meta_data import ExtendedMetaData
+from aviary.examples.external_subsystems.engine_NPSS.engine_variables import Aircraft, Dynamic
+from aviary.examples.external_subsystems.engine_NPSS.table_engine_connected_variables import vars_to_connect
+import pkg_resources
+import os
+import numpy as np
+
+
+class TableEngineBuilder(EngineModel):
+
+    def __init__(self, name='NPSS_prop_system'):
+        aviary_inputs = AviaryValues()
```

Comment:
This should probably be an input so the user doesn't have to use the default AviaryValues.
```suggestion
    def __init__(self, name='NPSS_prop_system', aviary_inputs = AviaryValues()):
```

Reply:
Fixed and uploaded.